guides/v2.2/frontend-dev-guide/layouts/xml-manage.md

```diff
@@ -345,56 +344,143 @@ This would make the product page look like following:
 To learn how to locate the layout file you need to customize, see [Locate templates, layouts, and styles].
 
 
-## Remove elements {#layout_markup_remove_elements}
+## Add functionality to existing elements {#layout_markup_add_to_elements}
 
-Elements are removed using the `remove` attribute for the `<referenceBlock>` and `<referenceContainer>`. 
+Let us say that we want to add functionality to a core template with custom logic using a ViewModel in the `cart/item/default.phtml` template found in `Magento/Checkout/view/frontend/layout/checkout_cart_item_renderers.xml`:
 
-**Example**: remove the Compare Products {% glossarytooltip 31751771-8163-434b-88bc-c5f94d859fc3 %}sidebar{% endglossarytooltip %} block from all store pages. 
+```xml
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+<body>
+    <referenceBlock cart.renderer="Magento/Checkout/view/frontend/layout/checkout_cart_item_renderers.xml">
```

Comment:
This is not valid XML: `cart.renderer="Magento/Checkout/view/frontend/layout/checkout_cart_item_renderers.xml"`. Consider replacing with: `name="checkout.cart.item.renderers.default"`